scanner/src/main/kotlin/scanners/fossid/FossIdScanResults.kt

```diff
@@ -33,6 +33,7 @@ import org.ossreviewtoolkit.model.Issue
 import org.ossreviewtoolkit.model.LicenseFinding
 import org.ossreviewtoolkit.model.PackageProvider
 import org.ossreviewtoolkit.model.RemoteArtifact
+import org.ossreviewtoolkit.model.Severity
```

Comment:
Commit message:
- Is the double "N" in "RETURNN" really correct?
- "alleviate" without "s" at the end
- If the middle section is supposed to be its own paragraph, please make it stand out by adding empty lines around it.

Reply:
>     * Is the double "N" in "RETURNN" really correct?

![image](https://github.com/oss-review-toolkit/ort/assets/66427221/3e2d87e5-2fb0-4d58-84c2-090ff22ebfff)
Yes